torch/testing/_comparison.py

```diff
@@ -1289,6 +1337,9 @@ def assert_close(
     :meth:`~torch.Tensor.qscheme` and the result of :meth:`~torch.Tensor.dequantize` is close according to the
     definition above.
 
+    If ``actual`` and ``expected`` are small floating point dtypes, they are compared bitwise. In other words,
+    ``rtol`` and ``atol`` are set to zero.
```

Comment:
I don't think we need this, this falls under the "other" category in the table? https://pytorch.org/docs/stable/testing.html